notes-service/src/main/java/org/exoplatform/wiki/jpa/entity/PageEntity.java

```diff
@@ -111,6 +111,9 @@ public class PageEntity extends BasePageEntity {
   @Column(name = "DELETED")
   private boolean deleted;
 
+  @Column(name = "TOBEPUBLISHED")
+  private boolean toBePublished;
```

Comment:
This modification doesn't seem fixing the real issue and the description doesn't mention the real cause of the problem and how did you analysed it and proposed this fix. We 'll have to re-work this together.